src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -131,19 +146,25 @@ Use modal for short and non-frequent tasks. For common tasks consider using the
         useHelperOpenModalButton({ title: "Default", setShow, openModalButtonRef }); // internal helper, not part of the API
       return (
         // Modal with default width variant
-        <>
+        <div className="check">
+          <div id="checkit" />
           {openModalButton}
           <Modal
             id="story-book-modal"
-            title="Modal title"
+            title="Modal titl11e"
             triggerElement={openModalButtonRef.current}
             show={show}
             onClose={closeModal}
             // Width prop effects on the modal width
             width={Modal.width.DEFAULT}
           >
-            <ModalContent>
-              <p>Modal content goes here</p>
+            <ModalContent id="modala">
+              <Dropdown
+                multi
+                defaultValue={[options[0], options[1]]}
+                options={options}
+                popupsContainerSelector={"#checkit"}
```

Comment:
let's fix IDs here to be more accurate? like not "modala", but "modal", not "checkit", but "modal-container" or smth like this

Reply:
for debug will remove